lightning/src/routing/router.rs

```diff
@@ -2276,7 +2275,7 @@ where L::Target: Logger {
 				panic!("first_hops should be filled in with usable channels, not pending ones");
```

Comment:
Could update the comment 10 lines up with the new last-hop processing we're doing